setup.py

```diff
@@ -39,16 +39,21 @@ def get_version():
     ],
     license='MIT',
     install_requires=[
-        'django>=1.6',
+        'coverage>=3.7.1',
+        'Django>=1.6.5,<1.8',
+        'jsonfield',
+        'six',
+        'psycopg2',
```

Comment:
Could you move this to be a test requirement? Although most users will probably use psycopg2, they may be on MySQL